model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -243,6 +244,21 @@ object ArrowInliner extends Logging {
     renamedArrows: Map[String, FuncArrow]
   )
 
+  given Monoid[AbilityResolvingResult] with
+
+    override val empty: AbilityResolvingResult =
+      AbilityResolvingResult(Map.empty, Map.empty, Map.empty)
+
+    override def combine(
+      a: AbilityResolvingResult,
+      b: AbilityResolvingResult
+    ): AbilityResolvingResult =
+      a.copy(
```

Comment:
```suggestion
      AbilityResolvingResult(
```
If new field will be added compiler will ensure this method will be updated